uwblueprint/social-diversity-for-children
# add initial signup form

Initial Signup Form
https://www.notion.so/uwblueprintexecs/Frontend-Initial-Sign-up-form-ca1f7c0f2d0448ccb7542c43de8c8d0a

Added initial signup form where user decides if they are a parent or user 

Steps to test

1. Run the app locally and head over to http://localhost:3000/en/signupform to see the new page
